101-jenkins-with-SSH-public-key/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "itemDisplayName": "Secure Jenkins on Azure (SSH public key authentication)",
```

Comment:
We still want to keep this as short as possible. I recommend "Jenkins on Azure (using ssh key)" and "Jenkins on Azure (using password)". You could get rid of the 'using' to make it even shorter if you want

And then I'd add it to the description:
"This template allows you to deploy a secure Jenkins instance on a DS1_v2 size Linux Ubuntu 14.04 LTS VM using an ssh key for authentication."